src/WebJobs.Script/Extensions/AssemblyExtensions.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Concurrent;
+using System.Reflection;
+
+namespace Microsoft.Azure.WebJobs.Script
+{
+    public static class AssemblyExtensions
+    {
+        private readonly static ConcurrentDictionary<Assembly, string> _codeBaseMapping = new ConcurrentDictionary<Assembly, string>();
```

Comment:
Introduced this since the we currently depend on the code base for resolution and that information won't be set to the assembly code base when loading raw bytes.
The code that depended on the code base has also been updated.